src/main/cpp/t07_ip.cpp

```diff
@@ -29,10 +29,34 @@
 
 #include "t07_ip.h"
 #include <iostream>
-
+#include <string>
+#include <cstdio>
 
 using namespace std;
 
 int t07_ip() {
-
+ string s = "";
+    string s_gen = "";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 's_gen' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4719848)